libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationRoute.java

```diff
@@ -282,8 +282,8 @@ public Builder alternatives(@Nullable Boolean alternatives) {
      * Languages</a>
      * @since 0.5.0
      */
-    public Builder language(String language) {
-      directionsBuilder.language(new Locale(language));
+    public Builder language(Locale language) {
```

Comment:
Could you reflect this change in the Javadoc ☝️?
```java
    /**
     * Set the instruction language for the directions request, the default is english. Only a
     * select number of languages are currently supported, reference the table provided in the see
     * link below.
     *
     * @param language a string value representing the language you'd like the instructions to be
     *                 written in when returned
     * @return this builder for chaining options together
     * @see <a href="https://www.mapbox.com/api-documentation/#instructions-languages">Supported
     * Languages</a>
     * @since 0.5.0
     */
```
This is not accurate anymore.